di-ipv-ticf-stub/lambdas/src/management/services/userEvidenceService.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  DynamoDB,
+  GetItemInput,
+  PutItemInput,
+  UpdateItemInput,
+} from "@aws-sdk/client-dynamodb";
+import { marshall, unmarshall } from "@aws-sdk/util-dynamodb";
+
+import { getSsmParameter } from "../../common/ssmParameter";
+import TicfEvidenceItem from "../../domain/ticfEvidenceItem";
+import UserEvidenceItem from "../model/userEvidenceItem";
+import { config } from "../../common/config";
+
+const dynamoClient = config.isLocalDev
+  ? new DynamoDB({
+      endpoint: config.localDynamoDbEndpoint,
+      region: config.region,
+    })
+  : new DynamoDB({ region: config.region });
+
+export async function persistUserEvidence(
+  userId: string,
+  ticfEvidenceItemReq: TicfEvidenceItem
+): Promise<void> {
+  const ticfStubTtl = getTtl(
+    await getSsmParameter(config.ticfParamBasePath + "ticfStubTtl")
+  );
+
+  const existingUserEvidence = await getUserEvidence(userId);
+  if (!existingUserEvidence) {
+    const userEvidence: UserEvidenceItem = {
+      userId: userId,
+      evidence: ticfEvidenceItemReq,
+      ttl: ticfStubTtl,
+    };
+    await saveUserEvidence(userEvidence);
+  } else {
+    await updateUserEvidence(userId, ticfEvidenceItemReq, ticfStubTtl);
```

Comment:
Do you actually need to update the existing record? I think we can just overwrite the existing record since all the fields are updated anyway.